detect_secrets/transformers/yaml.py

```diff
@@ -221,6 +221,7 @@ def _compose_node_shim(
         node.__line__ = line + 1
 
         if node.tag.endswith(':map'):
+            self.is_inline_flow_mapping_key = False
```

Comment:
Can you put some of your explanation from your PR description into here as a code comment?

Reply:
Updated